docs/howto/use-extensions/list-extensions.rst

```diff
@@ -0,0 +1,11 @@
+.. _list-extensions:
+
+List extensions
```

Comment:
Any value in linking to https://canonical-snapcraft--5212.com.readthedocs.build/en/5212/reference/commands/list-extensions/ ?

Reply:
Yes: [9d396a3](https://github.com/canonical/snapcraft/pull/5212/commits/9d396a3766d08519723e2e876a78ab1d65b325a6)